app/helpers/application_helper/toolbar/host_initiators_center.rb

```diff
@@ -0,0 +1,32 @@
+class ApplicationHelper::Toolbar::HostInitiatorsCenter < ApplicationHelper::Toolbar::Basic
+  button_group(
+    'host_initiator_vmdb',
+    [
+      select(
+        :host_initiator_vmdb_choice,
+        nil,
+        t = N_('Configuration'),
+        t,
+        :items => [
+          button(
+            :host_initiator_refresh,
+            'fa fa-refresh fa-lg',
+            N_('Refresh relationships and power states for all items related to these Host Initiators'),
+            N_('Refresh Relationships and Power States'),
+            :image   => "refresh",
+            :data    => {'function'      => 'sendDataWithRx',
+                         'function-data' => {:type => "refresh", :controller => "hostInitiatorToolbarController"}},
+            :confirm => N_("Refresh relationships and power states for all items related to these Host Initiators?"),
+            :options => {:feature => :refresh}
+          ),
+          button(
+            :host_initiator_new,
+            'pficon pficon-add-circle-o fa-lg',
+            t = N_('Define a new host initiator'),
```

Comment:
if you add `:url => "/new"` it will eliminate the need to go thru `button` method and you can remove https://github.com/ManageIQ/manageiq-ui-classic/pull/7587/files#diff-821df29f358b6837c673312f74ee982a7933ed4ee9de74236e46bb76ff8529ddR52-R54

Reply:
done